cfme/services/myservice/ssui.py

```diff
@@ -71,6 +71,7 @@ def is_displayed(self):
     )
     resource_power_status = PowerIcon(".//span/i[contains(@class, 'pficon') and contains("
                                       "@uib-tooltip,'Power State')]")
+    standard_output = Text('.//div[@class="col-lg-16"]')
```

Comment:
_Optional_ - If there is any other way to define this locator, I think that would be better, I don't really like using class value `col-lg-16` to point to this widet.

Reply:
I tried but I think we are good to go

Reply:
right this locator is risky